src/Spiffy.Monitoring/Config/Configuration.cs

```diff
@@ -27,5 +28,7 @@ public static void Initialize(Action<InitializationApi> customize)
         }
 
         internal static bool RemoveNewLines { get; private set; }
+
+        internal static int DeprioritizedValueLength { get; private set; } = 1024;
```

Comment:
I left this additional initialization in because it seems possible to use logging without `InitializationApi` and I wanted to have a sane default.  If I'm reading that wrong let me know and I'll remove it.

Reply:
interesting.  not sure why this class doesn't just keep an instance of InitializationApi, but you're following the pattern introduced by `RemoveNewLines` 👍